project/make/validate-toml

```diff
@@ -9,9 +9,7 @@ unset IFS
 badFiles=()
 for f in "${files[@]}"; do
 	# we use "git show" here to validate that what's committed is formatted
-	if [ "$(git show "$VALIDATE_HEAD:$f" | tomlv)" ]; then
-		badFiles+=( "$f" )
-	fi
+	tomlv $f || badFiles+=( "$f" )
```

Comment:
(ie, this needs to be `git show "$VALIDATE_HEAD:$f" | tomlv /proc/self/fd/0 || badFiles+=("$f")`)